src/tests/Test2/test2.addProductsToCart.spec.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable no-await-in-loop */
+/* eslint-disable indent */
+import { expect } from '@playwright/test';
+import { test } from '../../fixtures/base';
+
+test.describe('Test2 => Add several random Products to Cart and verify them', () => {
+    test('should login successfully, add products to cart and verify them', async (
+        /** @type {{ app: import('../pages/Application').Application }} */{ app },
+    ) => {
+        await app.login.navigate();
+        await app.login.performLogin('standard_user', 'secret_sauce');
+
+        await expect(app.inventory.headerTitle).toBeVisible();
+
+        const productCount = await app.inventory.inventoryItems.count();
+
+        const randomNumbers = new Set();
```

Comment:
Let's create a helper function to get a random product

Reply:
done